menpowidgets/base.py

```diff
@@ -793,6 +800,8 @@ def update_info(img, image_is_masked, group):
             text_per_line.append("> {} landmark points".format(
                 img.landmarks[group].lms.n_points))
             n_lines += 1
+        if custom_str is not None:
```

Comment:
You're actually checking if the provided custom function argument is ``None`` twice: before calling the `update_info` function (line 769) and inside the function (line 803). I think you should pass in the custom function directly as an argument in `update_info()`. Then the check will only happen once in the function and in case it is not ``None`` you can call it as `custom_info_callback(img)`.